packages/e2e-test-utils-playwright/src/editor/switch-block-inspector-tab.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Internal dependencies
+ */
+import type { Editor } from './index';
+import { expect } from '../test';
+
+/**
+ * Clicks on the block inspector tab button with the supplied label and waits
+ * for the tab switch.
+ *
+ * @param {Editor} this
+ * @param {string} label
+ */
+export async function switchBlockInspectorTab( this: Editor, label: string ) {
+	const inspectorTabButton = this.page.locator(
+		`role=region[name="Editor settings"i] >> role=tab[name="${ label }"i]`
+	);
+
+	if ( inspectorTabButton ) {
```

Comment:
I don't think this `if` will do anything, as a locator is (AFAIK) always truthy. It's like a black box around an element, it stores how to select the element without actually doing it.

In this code the element is only selected from the DOM at the point where `await locator.getAttribute( 'id' )` is called.

Reply:
✅ Done in https://github.com/WordPress/gutenberg/pull/47045/commits/175a868a0948eaa98e8854b254887e5704d61376